examples/jsm/renderers/CSS2DRenderer.js

```diff
@@ -118,7 +118,9 @@ class CSS2DRenderer {
 
 				const visible = ( object.visible === true ) && ( _vector.z >= - 1 && _vector.z <= 1 ) && ( object.layers.test( camera.layers ) === true );
 				object.element.style.display = ( visible === true ) ? '' : 'none';
-
+				object.element.classList.add('css-object');
```

Comment:
Can't you add these lines on application level instead? I'm not sure I understand why the renderer needs to be modified. The HTML elements of `CSS2DObject` are always created on app level.